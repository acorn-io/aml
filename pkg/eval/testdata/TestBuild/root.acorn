a: {
	a: $.a.b
	b: 1
}