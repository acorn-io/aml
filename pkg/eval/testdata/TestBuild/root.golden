&eval.File{Body: &eval.Struct{
	Position: eval.Position{
		Filename: "root.acorn",
		Line:     1,
		Column:   1,
	},
	Fields: []eval.Field{&eval.KeyValue{
		Key: eval.FieldKey{
			Key: eval.Value{
				Value: value.String("a"),
			},
			Pos: eval.Position{
				Filename: "root.acorn",
				Line:     1,
				Column:   1,
			},
		},
		Value: &eval.Struct{
			Position: eval.Position{
				Filename: "root.acorn",
				Offset:   3,
				Line:     1,
				Column:   4,
			},
			Fields: []eval.Field{
				&eval.KeyValue{
					Key: eval.FieldKey{
						Key: eval.Value{Value: value.String("a")},
						Pos: eval.Position{
							Filename: "root.acorn",
							Offset:   6,
							Line:     2,
							Column:   2,
						},
					},
					Value: &eval.Selector{
						Pos: eval.Position{
							Filename: "root.acorn",
							Offset:   9,
							Line:     2,
							Column:   5,
						},
						Base: &eval.Selector{
							Pos: eval.Position{
								Filename: "root.acorn",
								Offset:   9,
								Line:     2,
								Column:   5,
							},
							Base: &eval.Lookup{
								Pos: eval.Position{
									Filename: "root.acorn",
									Offset:   9,
									Line:     2,
									Column:   5,
								},
								Key: "$",
							},
							Key: eval.Value{Value: value.String("a")},
						},
						Key: eval.Value{Value: value.String("b")},
					},
					Pos: eval.Position{
						Filename: "root.acorn",
						Offset:   6,
						Line:     2,
						Column:   2,
					},
				},
				&eval.KeyValue{
					Key: eval.FieldKey{
						Key: eval.Value{Value: value.String("b")},
						Pos: eval.Position{
							Filename: "root.acorn",
							Offset:   16,
							Line:     3,
							Column:   2,
						},
					},
					Value: eval.Value{Value: value.Number("1")},
					Pos: eval.Position{
						Filename: "root.acorn",
						Offset:   16,
						Line:     3,
						Column:   2,
					},
				},
			},
		},
		Pos: eval.Position{
			Filename: "root.acorn",
			Line:     1,
			Column:   1,
		},
	}},
}}
